src/prompts.ts

```diff
@@ -76,10 +98,9 @@ export const prompts = {
                 name: 'network',
                 message: `${colorCodes.magentaColor}Which network do you want to connect to?${colorCodes.resetColor}`,
                 choices: [`Flare ${colorCodes.greenColor}(Mainnet)`, `Coston2 ${colorCodes.yellowColor}(Testnet)`],
-                filter: function (val: string) {
+                filter: (val: string) => {
                     const network = val.split(" ")[0]
-                    if (network == "flare") { return "flare" }
-                    else return "costwo"
+                    return network == "flare"? "flare" : "costwo"
```

Comment:
can you create the constant for the networks? it will be easier to manage, modify.